mods/d2k/maps/atreides-04/atreides04-AI.lua

```diff
@@ -0,0 +1,163 @@
+IdlingUnits = { }
+
+AttackGroupSize =
+{
+	easy = 6,
+	normal = 8,
+	hard = 10
+}
+
+AttackDelays =
+{
+	easy = { DateTime.Seconds(4), DateTime.Seconds(7) },
+	normal = { DateTime.Seconds(2), DateTime.Seconds(5) },
+	hard = { DateTime.Seconds(1), DateTime.Seconds(3) }
+}
+
+HarkonnenInfantryTypes = { "light_inf", "light_inf", "trooper", "trooper", "trooper" }
+HarkonnenVehicleType = { "combat_tank_h" }
+
+AttackOnGoing = false
```

Comment:
Renamed it. Also `Attacking = false` is not needed. In lua there's not really a difference between a not (yet) existing boolean and a boolean with the value of "false". I.e.
```
if not alarm then
```
would be the same as
```
alarm = false
if not alarm then
```
I don't remember why I added those `AttackOnGoing = false` and `HoldProduction = false` in the first place. Probably to remember myself of something and then I forgot to remove then (which I did now).